modAionImpl/test/org/aion/zero/impl/consensus/BalanceTransferConsensusTest.java

```diff
@@ -43,6 +48,17 @@
 
     private StandaloneBlockchain blockchain;
 
+    private static byte txType;
+
+    @Parameters
+    public static Object[] data() {
+        return new Object[] {TransactionTypes.DEFAULT, TransactionTypes.AVM_CREATE_CODE};
+    }
+
+    public BalanceTransferConsensusTest(byte _txType) {
```

Comment:
I don't see where this gets instantiated and I'm not convinced the transaction type is actually changed.
I think it would be better if the tests were parametrized instead when needed.